src/logic/calculate.js

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable */
```

Comment:
- [ ] `required`
- [ ] Amazing work so far 👏 
- [ ] However, you disabled the eslint, please enable it. cause it defeats the purpose of installing the linter to detect errors